pkgs/stdenv/generic/setup.sh

```diff
@@ -422,50 +421,52 @@ fi
 
 
 substitute() {
-    local input="$1"
-    local output="$2"
+    local input=$1; shift
+    local output=$1; shift
```

Comment:
`shift` also takes a numeric argument, so you can do e.g., `local foo=$1 bar=$2; shift 2` to achieve the same effect.